src/views/Alteration.vue

```diff
@@ -48,12 +48,14 @@
 
       <documents-delivery
        class="mt-10"
+       :validate="getAppValidate"
        :pleaseValidate="true"
```

Comment:
Is this prop now obsolete?

Also on line 58.

Reply:
So, when the file and pay button is clicked, getAppValidate is set to true, triggering the component validation. Cam set it up for the court order stuff

Reply:
What about `pleaseValidate` ?

Reply:
That is hard coded to true and does nothing yet :) Should I remove?

Reply:
Yes, please remove this. It was the initial validation implementation, now replaced by `getAppValidate`.

Reply:
Done